src/components/LeaderListItem/index.jsx

```diff
@@ -14,12 +14,18 @@ const LeaderListItem = props => {
       <span className="placement">{props.placement}</span>
       <Avatar size={40} src={props.image} />
 
-      <span className="name">
-        <Link to={'/profile/' + props.uuid}>{props.name}</Link>
+      <span className="column left">
+        <div className="name">
+          <Link to={'/profile/' + props.uuid}>{props.name}</Link>
```

Comment:
can we add proptypes for the uuid it never got implemented. should just be a string

Reply:
Yup, just added it in.